### PR TITLE
Run periodic update jobs to update envoyproxy/envoy v1.19 branch

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

File: Makefile.core.mk [hunk 1]
```diff
@@ -31,6 +31,7 @@ generate-config:
 	@(cd prow/config/cmd; go run generate.go write)
 	@rm -fr prow/cluster/jobs/istio-private/*/*.gen.yaml
 	@go run prow/genjobs/main.go --configs=./prow/config/istio-private_jobs
+	@go run prow/genjobs/main.go --configs=./prow/config/istio-private_jobs
```
Comment (jwendell) [inline Makefile.core.mk]: why duplicate this?

File: prow/config/jobs/envoy-1.11.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- name: update-proxy
+  types: [periodic]
+  interval: 24h
+  command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=proxy
+  - "--title=Automator: update envoy@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH"
+  - --labels=auto-merge
```
Comment (jwendell) [inline prow/config/jobs/envoy-1.11.yaml:49]: Not sure if we want automatic stuff being merged in a micro release...

**Review by jwendell — CHANGES_REQUESTED** [review]

File: Makefile.core.mk (hunk 1 above)
Comment (jacob-delgado) [reply Makefile.core.mk]: Fixed. I was going to remove the `istio-private.envoy` file. Decided not to remove it.

File: prow/config/jobs/envoy-1.11.yaml (hunk 2 above)
Comment (jacob-delgado) [reply prow/config/jobs/envoy-1.11.yaml:49]: We do for 1.10. The @istio/wg-networking-maintainers group wanted Istio to no longer have to maintain istio/envoy.

**Comment by ericvn** [comment]
Slightly different way in https://github.com/istio/test-infra/pull/3521 where I moved the job to proxy instead of using envoy. Not sure which way is better.

**Comment by jacob-delgado** [comment]
Closed in lieu of Eric's PR

**Pull request closed without merging**